hack/release.go

```diff
@@ -23,36 +23,55 @@ import (
 	"log"
 	"os"
 	"path/filepath"
+	"reflect"
+	"sort"
 	"strings"
 	"text/template"
 
 	"sigs.k8s.io/yaml"
 
 	mellanoxv1alpha1 "github.com/Mellanox/network-operator/api/v1alpha1"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
 )
 
+// ReleaseImageSpec contains ImageSpec in addition with Image SHA256.
+type ReleaseImageSpec struct {
+	Shas []SHA256
+	mellanoxv1alpha1.ImageSpec
+}
+
+// SHA256 contains container image in sha256 format and a description.
+type SHA256 struct {
+	Sha  string
```

Comment:
sha is the image reference in "sha format"  e.g   `some.rep/project/image-repo@sha256:abcdef`

maybe rename: to SHAImageRef ? or ImageRef  ?  since its already in sha256 struct

Reply:
Done